.travis.yml

```diff
@@ -8,20 +8,21 @@ env:
     - WORKSPACE="Example/GiniVision.xcworkspace"
     - IOS_FRAMEWORK_SCHEME="GiniVision-Example"
     - IOS_SDK=iphonesimulator10.0
-    - IOS_SDK_LEGACY=iphonesimulator9.3
 matrix:
   include:
-    - osx_image: xcode7.3
-      env: DESTINATION="OS=8.1,name=iPhone 4S"          SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK_LEGACY"     RUN_TESTS="NO"      POD_LINT="YES"     BUILD_DOCS="YES"     BUILD_COVERAGE="NO"
-    - osx_image: xcode7.3
-      env: DESTINATION="OS=8.3,name=iPhone 5S"          SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK_LEGACY"     RUN_TESTS="NO"      POD_LINT="NO"      BUILD_DOCS="NO"      BUILD_COVERAGE="NO"
-    - osx_image: xcode7.3
-      env: DESTINATION="OS=9.1,name=iPhone 6 Plus"      SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK_LEGACY"     RUN_TESTS="YES"     POD_LINT="NO"      BUILD_DOCS="NO"      BUILD_COVERAGE="NO"
-    - osx_image: xcode7.3
-      env: DESTINATION="OS=9.3,name=iPhone 6S"          SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK_LEGACY"     RUN_TESTS="YES"     POD_LINT="NO"      BUILD_DOCS="NO"      BUILD_COVERAGE="YES"
+    - osx_image: xcode8
+      env: DESTINATION="OS=8.1,name=iPhone 4S"          SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK"     RUN_TESTS="NO"      POD_LINT="YES"     BUILD_DOCS="YES"     BUILD_COVERAGE="NO"
+    - osx_image: xcode8
+      env: DESTINATION="OS=9.1,name=iPhone 5S"          SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK"     RUN_TESTS="NO"      POD_LINT="NO"      BUILD_DOCS="NO"      BUILD_COVERAGE="NO"
+    - osx_image: xcode8
+      env: DESTINATION="OS=9.3,name=iPhone 6 Plus"      SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK"     RUN_TESTS="YES"     POD_LINT="NO"      BUILD_DOCS="NO"      BUILD_COVERAGE="NO"
+    - osx_image: xcode8
+      env: DESTINATION="OS=10.0,name=iPhone 6S"         SCHEME="$IOS_FRAMEWORK_SCHEME"     SDK="$IOS_SDK"     RUN_TESTS="YES"     POD_LINT="NO"      BUILD_DOCS="NO"      BUILD_COVERAGE="YES"
 before_install:
   - gem install slather
-  - gem install cocoapods # Since Travis is not always on latest version
+   # Pre-Release version of Cocoapods needed for Xcode8 environment.
+   # Can be removed when v1.1.0 is officially released.
+  - gem install cocoapods --pre # Since Travis is not always on latest version
```

Comment:
We can now remove `-- pre` and the comments as Cocoapods has been updated to version 1.1.1.